src/contexts/TradeTableContext.tsx

```diff
@@ -56,6 +59,8 @@ export const TradeTableContextProvider = (props: {
     const [selectedOutsideTab, setSelectedOutsideTab] = useState(0);
     const [outsideControl, setOutsideControl] = useState(false);
 
+    const CHART_MIN_HEIGHT = 4;
```

Comment:
this seems to be duplicated in ChartContext as well... is there somewhere central where we can consolidate this?